api/Service/TokenService.cs

```diff
@@ -0,0 +1,51 @@
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Text;
+using api.Interfaces;
+using api.Models;
+using Microsoft.IdentityModel.Tokens;
+
+namespace api.Service;
+
+public class TokenService : ITokenService{
+    private readonly IConfiguration _config;
+    private readonly SymmetricSecurityKey _key;
+
+    public TokenService(IConfiguration config){
```

Comment:
```suggestion
    public TokenService(IConfiguration config) {
```

В продуктах JetBrains форматировать кода по код-стилю доступно с помощью `Ctrl + Alt + L`. Используй это. Если пишешь в Visual Studio - то погляди какая комбинация клавиш там есть.

Тоже самое касается импортов. В Rider это `Ctrl + Alt + O`. Используй и не допускай лишних импортов в файле

Reply:
Я не сделал себе лицензию, поэтому в VS code пишу. Надо действительно посмотреть как форматировать в VS

Reply:
Пока что никак. Есть расширение code formatter, но он только для js